inode/config.go

```diff
@@ -121,33 +121,36 @@ func startInodeCacheDiscard(confMap conf.ConfMap, volume *volumeStruct, volumeSe
 	)
 	LRUCacheMaxBytes, err = confMap.FetchOptionValueUint64(volumeSectionName, "MaxBytesInodeCache")
 	if nil != err {
-		return
+		LRUCacheMaxBytes = 10485760 // TODO - Remove setting a default value
+		err = nil
 	}
 	volume.inodeCacheLRUMaxBytes = LRUCacheMaxBytes
 
 	LRUDiscardTimeInterval, err = confMap.FetchOptionValueDuration(volumeSectionName, "InodeCacheEvictInterval")
 	if nil != err {
-		return
+		LRUDiscardTimeInterval = 1 * time.Second // TODO - Remove setting a default value
+		err = nil
 	}
 
-	if LRUDiscardTimeInterval == 0 {
+	if LRUDiscardTimeInterval != 0 {
```

Comment:
I guess we technically shouldn't assume 0s is == 0... time.Duration just happens to be an int64 and I gather the parser is being friendly and not requiring us to type cast 0 to be either int64(0) or, more completely, time.Duration(0).